Stratis.Bitcoin.Features.Wallet/Interfaces/IWalletManager.cs

```diff
@@ -112,6 +112,13 @@ public interface IWalletManager
         IEnumerable<HdAddress> GetHistory(string walletName);
 
         /// <summary>
+        /// Return a flat representation of the wallet history.
+        /// </summary>
+        /// <param name="walletName">The wallet name.</param>
+        /// <returns>The history in a address to transaction pairs.</returns>
```

Comment:
Not sure a better alternative by I find this comment confusing. Maybe this "Collection of address history and transaction pairs."?

Reply:
Ha yeah totally confusing now that I read it yap I will fix